src/Symfony/Component/ErrorRenderer/composer.json

```diff
@@ -33,6 +33,7 @@
     },
     "autoload": {
         "psr-4": { "Symfony\\Component\\ErrorRenderer\\": "" },
+        "classmap": [ "Resources/stubs/Exception/FlattenException.php" ],
```

Comment:
this would fix the compiler error mentioned here https://github.com/symfony/symfony/issues/33929#issuecomment-539985732